README.md

```diff
@@ -245,16 +245,24 @@ Note that you can also use the [Rendertron middleware](https://github.com/Google
     "start": "prpl-server --host 0.0.0.0 --https-redirect"
```

Comment:
Weren't there some issues with how you launch the server from standard? They got fixed and this just works now I guess?

Reply:
Yes, you can now launch with npm start command (don't need server.js).

Reply:
nice!